### ISSUE TITLE
CRM Deleting Email Relationships without permission

### ISSUE THREAD

**Issue by appomattox** [salesagility/SuiteCRM#2077]
**Summary**
Whenever I import leads the CRM deletes the relationship between the lead and its primary email.

#### Issue
By default, the CRM does not establish links between Accounts and Leads, I don't know why, because there's a field (account_id) so that lead names will show up in the account record detail's subpanel. As a result, I've had to create a custom field (account_id_c) and relate it to the Accounts module. However, that doesn't complete the link where I can link back and forth between a lead and its account. Account_id_c doesn't feed into the Lead subpanel in the record's detail view. As a result, I've had to write a scheduled job script that checks the account_id and account_id_c values to see if they're equal, which they won't be if they're imported (account names are typically included, but very rarely do they have accompanying account IDs).

#### Expected Behavior
The a link between a lead and an account should be established. Nothing more.

#### Actual Behavior
The CRM writes 1 in the "deleted field" of the email_addr_bean_rel table without my permission when the sceduled job runs. There's nothing in the script for the scheduled job that interacts with the lead's email address, and I don't know why changing the account_id or account_id_c values have anything to do with email addresses. 

#### Possible Fix
Is there anything specifying that a relationship should be broken in the source code anywhere if account_id is changed? 

#### Steps to Reproduce

1. Create a custom field in studio and relate it to Accounts module.
2. Build a scheduled job that sets the record's account_id and account_id to be the same and set it for any interval you like.
3. Import leads with emails and wait.

#### Your Environment

* SuiteCRM Version used: 7.7
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Any
* Environment name and version (e.g. MySQL, PHP 7): MySQL v5.5.5.cii,  PHP v5.5.35
* Operating System and version (e.g Ubuntu 16.04): CENT OS

**Comment by samus-aran** [comment]
Unfortunately we can't raise a bug with user's custom code (i.e. the scheduler job).  Perhaps focus this issue on the actual importer scenario and we'll try and replicate this.  I would recommend you edit this issue to reflect that or raise your scheduler job query in the forums. [https://suitecrm.com/forum](url)

**Comment by appomattox** [comment]
Thanks for getting back with me. Your answer was as I suspected. However, I have also posted this question in the forum ([https://suitecrm.com/forum/suitecrm-7-0-discussion/10922-conditions-for-breaking-relationship-between-lead-and-primary-email](https://suitecrm.com/forum/suitecrm-7-0-discussion/10922-conditions-for-breaking-relationship-between-lead-and-primary-email)) and also in Stack Overflow ([http://stackoverflow.com/questions/39234684/does-saving-a-bean-delete-the-records-connection-to-it-email-in-suitecrm](http://stackoverflow.com/questions/39234684/does-saving-a-bean-delete-the-records-connection-to-it-email-in-suitecrm)). I have not received an answer just yet.
